docs/sources/whatsnew/whats-new-in-v8-0.md

```diff
@@ -243,3 +243,7 @@ Documentation was updated to reflect these changes.
 ### Elasticsearch: Use application/x-ndjson content type for multi-search requests
 
 For multi-search requests, we now use the correct application/x-ndjson content type instead of the incorrect application/json. Although this should be transparent to most of the users, if you are running Elasticsearch behind a proxy, then be sure that your proxy correctly handles requests with this content type.
+
+### Prometheus: Change default http method to POST
```

Comment:
@achatterjee-grafana This is only new change to What's new, could you please review it.

Reply:
We mentioned this in 7.5 (https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v7-5/#changed-default-http-method-for-new-prometheus-data-sources). Should we mention this again?

Reply:
It is different. Previously we just set httpMethod to POST for new instances. This is going to change for all instances that don't have specifically selected GET/POST method - that left the http method to be default.

Reply:
maybe we should change the title : `Change default http method to POST for existing datasources`

Reply:
Yeah, I like that. Updated it so it is more clear.

Reply:
Update default HTTP method to POST for existing data sources